frontend/src/stores/roles/state.ts

```diff
@@ -1,6 +1,11 @@
 import type { Role } from '@/domain/Role'
+import type { Level } from '@/domain/Level'
 import { type Ref, ref } from 'vue'
 
 const roles: Ref<Role[]> = ref([])
 
 export { roles }
+
+const levels: Ref<Level[]> = ref([])
```

Comment:
Levels state should not exist, the levels inside the roles should be the only information to be considered